include/logging/log_ctrl.h

```diff
@@ -66,6 +66,8 @@ int log_set_timestamp_func(timestamp_get_t timestamp_getter, u32_t freq);
 /**
  * @brief Switch logger subsystem to panic mode.
  *
+ * Returns immediately if logger already in panic mode.
+ *
  * @details On panic logger subsystem informs all backends about panic mode.
```

Comment:
If you feel like correcting also this line: "On panic+, the+ logger [..]"

Reply:
done